wdfn-server/config.py

```diff
@@ -26,7 +26,7 @@
 
 DEBUG = False
 
-SERVER_SERVICE_ROOT = 'https://waterservices.usgs.gov'  # Used for webserver calls to waterservices. Allows us to use a private url.
+SERVER_SERVICE_ROOT = 'https://waterservices.usgs.gov'  # Used for webserver calls to waterservices.
 SERVICE_ROOT = 'https://waterservices.usgs.gov'  # Use for client side calls to waterservices. Most be a public url.
```

Comment:
Confused by why SERVER_SERVICE_ROOT and SERVICE_ROOT have the same value

Reply:
So the server one is server side as the name implies and the comment states whereas SERVICE_ROOT is for client side. This allows us to use internal only URLs for fetches performed by the server which may have performance benefits. The client side (at least for the public) must be a publicly available URL.

Reply:
Hmm. So https://waterservices.usgs.gov has both internal and external URLS? I thought the external vs internal was a property set at the domain level.

Reply:
By default we are using the same URL (makes local development easier). But in production, these are not the same.